source/_integrations/ecoforest.markdown

```diff
@@ -0,0 +1,65 @@
+---
+title: Ecoforest
+description: Instructions on how to integrate Ecoforest fireplaces with Home Assistant.
+ha_category:
+  - Climate
+ha_release: 2022.9
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@pjanuario'
+ha_domain: ecoforest
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The Ecoforest integration allows monitoring and control of local [Ecoforest](https://ecoforest.com) fireplaces in Home Assistant. These fireplaces are manufactured in Spain.
+
+There is currently support for the following device platforms within Home Assistant:
+
+- [Sensor](#sensor)
+
+{% include integrations/config_flow.md discovery=False %}
+
+<div class='note'>
+Devices must first be bound to WiFi or LAN using the app or website provided by the device manufacturer. It is not possible to connect devices to WiFi using this integration.
```

Comment:
```suggestion
Devices must first be connected to the internet using the app or website provided by the device manufacturer. It is not possible to connect devices to the internet using this integration.
```

Reply:
Well, in this case it is local device pooling so no internet access is needed. My devices don't have internet access just local network.

Reply:
right, in that case WiFi -> Wi-Fi